### ISSUE TITLE
Suggestings for R interface

### ISSUE THREAD

**Issue by abalter** [mamba-org/mamba#2506]
TL;DR---to make an R interface, should I pass the standard commands to the system, or wrap lower-level functions?

I'm taking some small steps to make an R interface for `*da`. I already have a function I use to switch environments on the fly by setting paths and environment variables. 

I'm working on another for when we need to install an R package that doesn't have a recipe. Because `install.packages` installs all the dependencies as well, I first go to cran and find the dependencies and make sure all of those are installed with conda (working backwards iteratively as necessary). I plan to use the R `pkgsearch` package to do:

```r
dependencies = 
    cran_package_history("pkgsearch") %>% 
    select(dependencies) %>% 
    unnest(cols="dependencies") %>% 
    filter(type=="Imports") %>% 
    pull(package)
```

A complete interface would wrap all `*da` commands. Would it be good enough to just use the regular CLI as system calls? Or would it be better to wrap `*da` functionality at a lower level? Is there even such a thing?

**Comment by wolfv** [comment]
@abalter there is a simple R wrapper (that hasn't been updated in a while) here: https://github.com/mamba-org/rhumba

If you want to revive this, that would be awesome!
